common/common.cpp

```diff
@@ -191,6 +191,15 @@ int32_t cpu_get_num_math() {
 // CLI argument parsing
 //
 
+static std::string get_cached_file(const std::string & path) {
```

Comment:
For clarification, I'd suggest changing the name and argument of this function to `fs_get_cache_file_path(const std::string & url_path)` and place it near other `fs_*` functions.

Reply:
Moved & renamed to fs_get_cache_file(filename), to make semantics clearer (caller now has to extract the filename).